Formula/qt.rb

```diff
@@ -136,14 +135,6 @@ class Qt < Formula
     directory "qtbase"
   end
 
-  # Fix Linux build with CMake versions >= 3.25
-  # remove in next release
-  patch do
-    url "https://github.com/qt/qtwebengine/commit/240e71877865ed07e4c8d5bd4553aa0772c2adf4.patch?full_index=1"
-    sha256 "8fb13bfc7aac50084e1c533955564a1819bbb25b544ebccd05b99e24527c7b80"
-    directory "qtwebengine"
-  end
-
   def install
     python = "python3.10"
```

Comment:
May be able to update to `python@3.11` as I think the backport commit is in this release.
- https://github.com/qt/qtwebengine/tree/v6.4.2/src 
- https://github.com/qt/qtwebengine-chromium/tree/0a34013e271bd8831255bb01c9c8284459d17cca

Should be able to supersede #119090 if it works.

Reply:
Let's give it a shot!

Reply:
```suggestion
    python = "python3.11"
```